website/source/docs/providers/github/r/team_membership.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "github"
+page_title: "Github: github_team_membership"
+sidebar_current: "docs-github-resource-team-membership"
+description: |-
+  Provides a Github team membership resource.
+---
+
+# github\_team_membership
+
+Provides a Github team membership resource.
+
+This resource allows you to add/remove users from teams in your organization. When applied,
+the user will be added to the team. If the user hasn't accepted their invitation to the 
+organization, they won't be part of the team until they do. When
+destroyed, the user will be removed from the team.
+
+## Example Usage
+
+```
+# Add a user to the organization
+resource "github_membership" "membership_for_some_user" {
+    username = "SomeUser"
+    role = "member"
+}
+
+resource "github_team" "some_team" {
+	name = "SomeTeam"
+	description = "Some cool team"
+}
+
+resource "github_team_membership" "some_team_membership" {
+	team_id = "${github_team.some_team.id}"
+	username = "SomeUser"
+	role = "member"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `team_id` - (Required) The Github team id
+* `username` - (Required) The user to add to the organization.
+* `role` - (Required) The role of the user within the team. 
```

Comment:
I think we should default this to member. Perhaps do the same for organization membership. Default them to the lowest privilege.